section/authentication.html

```diff
@@ -0,0 +1,302 @@
+<section id="authentication">
+  <h2>Authentication</h2>
+  <p>This section defines the key material and requirements for generating and managing cryptographic keys used to
+    create HTTP message signatures for Open Payments.</p>
+
+  <p>All authenticated Open Payments protocol operations initiated by the user agent and sent to either
+    the user's wallet provider or a website's wallet provider
+    (e.g. grant requests, quote creation, outgoing payment creation) MUST be signed.</p>
+
+  <p>The validity of signatures is established because the user, during setup or an authorization procedure,
+    registers the base64-encoded public JWK with their wallet provider. The wallet provider then uses
+    that public key to verify subsequent signed requests (for example, grant creation, quote requests,
+    and outgoing payment creation).</p>
+
+  <p class="note" title="Privacy">
+    Reusing a single long-lived signing key across
+    multiple origins or wallet providers can enable cross-site tracking.
+    Future revisions may define per-session or per-origin derived authentication
+    keys to further reduce tracking surfaces. See the <a href="https://docs.google.com/document/d/1s9aXcLoBdLe9y_8JispFK7NVERxUhxn0orqgzZBrcBo/edit?tab=t.0#heading=h.z0cxd4l694ag">
+    directed identity proposal</a> for background.
+  </p>
+
+  <p>User agents MUST use an [=Ed25519 key pair=] to sign HTTP requests that
+    require proof-of-possession as described in this specification.</p>
+
+  <p>A generated key pair is persisted in {{Storage}}.</p>
+
+  <p class="note">The user agent SHOULD display the base64-encoded public JWK on its [=setup page=] to help
+      the user register it with their wallet provider.</p>
+
+  <p class="note">
+    Regenerating the key pair invalidates the association the user previously
+    made between their public key and their wallet. The user would have to
+    re-upload or re-register the new public key and repeat any interactive
+    authorization flows.
+  </p>
+
+  <section>
+    <h3 id="ed25519-key-pair">Ed25519 key pair</h3>
+    <p>The <dfn data-lt="key id|kid">Ed25519 key id</dfn> (<code>kid</code>) is a globally unique opaque
+      identifier bound 1:1 to an [=Ed25519 key pair=].</p>
```

Comment:
The globally unique is not a requirement in case. It just has to be unique for given wallet address. Using 1,2,3 as `kid` are allowed as well.